src/test/llmnode_test.cpp

```diff
@@ -184,7 +183,7 @@ TEST_F(LLMOptionsKfsTest, DISABLED_LLMNodeOptionsCheckDefault) {
             output_stream: "RESPONSE:out1"
             node_options: {
                 [type.googleapis.com / mediapipe.LLMCalculatorOptions]: {
-                    models_path: "/workspace/"
+                    models_path: "/llm_testing/facebook/opt-125m"
```

Comment:
It would be the best if it works also in interactive sessions based on developer guide. Right now it will not.
option1 - update docker run command in developer guide to add mounting /llm_testing
option2 - use default path in the git repo and use /ovms/llm_testing path instead.

Reply:
going with option 2, so we don't expand container setup